### DISCUSSION TITLE
How to convert my Djinni project to this?

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [pydjinni/pydjinni#2] [General]
Hi,

How to convert my Djinni project to this?

https://github.com/nativium/nativium

Thanks.

**Comment by jothepro** [comment]
There is a brief documentation on what you might need to consider for migrating to PyDjinni [here](https://pydjinni.github.io/pydjinni/latest/migration/).

As it seems like Nativium is written in Python, you might want to experiment with the new [Python API](https://pydjinni.github.io/pydjinni/latest/api/).

Please keep in mind that the project is still in early alpha! The CLI, API and IDL might still be subject to major changes and I highly recommend to use the generated code with caution!

**Reply by paulocoutinhox** [reply]
Hi @jothepro,

Thanks for your efforts in this project, that is really nice!

Do you think that is time to replace original Dijnni?

I have an old branch here to use your project.

Thanks.
